core/server/api/canary/utils/validators/input/schemas/members-upload.json

```diff
@@ -0,0 +1,49 @@
+
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "members.upload",
+    "title": "members.upload",
+    "description": "Schema for members.upload",
+    "type": "array",
+    "items": {
+        "type": "object",
+        "additionalProperties": true,
+        "required": ["email"],
+        "properties": {
+            "name": {
+                "type": "string",
+                "maxLength": 191,
+                "pattern": "^([^,]|$)"
+            },
+            "email": {
+                "type": "string",
+                "minLength": 1,
+                "maxLength": 191,
+                "format": "email"
+            },
+            "note": {
+                "type": "string",
+                "minLength": 0,
+                "maxLength": 2000
+            },
+            "subscribed": {
+                "type": ["string"],
+                "enum": ["true", "false", "TRUE", "FALSE", ""]
```

Comment:
These boolean values come in as strings at this stage - they get converted in the importer itself. Chose this as the most close to real world way to see if the value is boolean.